README.md

```diff
@@ -130,7 +130,21 @@ At different positions inside the data you need to specify a country or a citize
         "kommentare": [ String ]
     }
 
-The property *kundenbetreuer.partnerId* is mandatory.
+The property *kundenbetreuer.partnerId* is mandatory.\
+Concerning *leadquelle*:
+
+* If you're using the selfservice via api (RaaS) please provide the correct value for “leadquelle” for your selfservice-use case. That is important in order to generate appropriate and valuable reporting.
```

Comment:
What is "selfservice via api (RaaS)"? How do you know that you are using this?

Reply:
So steht es in der API-Dokumentation. Und wenn API-Nutzer:innen die APIs angebunden haben nehme ich an, sie haben die Doku gelesen und sollten das kennen. Oder wir sprechen von "If you're using the Ratenkredit as a Service (RaaS)..."